.github/workflows/vercel-deploy.yml

```
@@ -73,36 +85,36 @@ jobs:
         echo "url=$OUTPUT" >> $GITHUB_OUTPUT
         echo "🚀 Deployed to: $OUTPUT"
       working-directory: ./web
-      
+
     - name: Comment on PR
       if: github.event_name == 'pull_request' && steps.check-token.outputs.skip != 'true'
       uses: actions/github-script@v7
       with:
         script: |
```

actionlint (1.7.11)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)